platform-validation/docs/aziot-compatibility-get-started.md

```diff
@@ -52,21 +52,25 @@ Following are the checks that are performed in the Platform Compatibility script
  
  * To download the script, use the following command
  
-`wget aka.ms/aziot-compat-prev`
+`wget aka.ms/aziot-compatibility.sh`
+
+* To make script executable, use the following command
+ 
+`chmod +x ./aziot-compatibility.sh`
 
 * To run the script, use the following commands for help.
 
-`./aziot-compatibilty --help` or `./aziot-compatibilty -h`
+`./aziot-compatibilty.sh --help` or `./aziot-compatibilty.sh -h`
```

Comment:
"compatibility" is missing the last "i" in several instances here

Reply:
good catch!

Reply:
Thanks, I will update the aka.ms link

Reply:
Ack Comments